pynxtools/dataconverter/readers/sts/__init__.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""
+    To collect data from Bias Spectroscopy output file that is mainly a
+    file with dat extension.
+"""
+# -*- coding: utf-8 -*-
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# To mvake the functions available in stm module
+# pylint: disable=unused-import
+from pynxtools.dataconverter.readers.sts.sts_file_parser import get_sts_raw_file_info
+from pynxtools.dataconverter.readers.sts.stm_file_parser import get_stm_raw_file_info
```

Comment:
the reader name is still sts. Do we support sts and/or stm. Is sts the best choice for the name of the reader?